delocate/libsana.py

```diff
@@ -24,6 +31,9 @@ def tree_libs(start_path, filt_func=None):
         If None, inspect all files for library dependencies. If callable,
         accepts filename as argument, returns True if we should inspect the
         file, False otherwise.
+    ignore_missing : bool
+        Determines if dependency resolution failures are considered a critical
+        error to be raised.  Otherwise ignored libraries are only printed out.
```

Comment:
Could you also add a `Raises` section at the end for the `DependencyNotFound` error?  <https://numpydoc.readthedocs.io/en/latest/format.html>

Reply:
I almost added a raises section but I didn't find anything else in the module to reference when I looked and didn't notice this was using the NumPy docstring style guide.  I'll add one now.

A contributing file might have helped.